### DISCUSSION TITLE
My cash from gameing

### DISCUSSION THREAD

**Discussion by Tacety** [blockscout/blockscout#9150] [General]
This is my account Tyler Autry Deneault tdogjake76@gmail.com 
12505742086 
Every body hear is trying to steel my money and all of all my nfts and whatever I see on my file I never gave no permissions to copy or clon or any transfers of any of my money or money cards or and and all of these transactions ha e been done iligaly to my acounrmt by people I don't even know the last game I just played I was saposto win 1000 dollars so when I logged in my email the game at the end for som reason switched from English to Spanish and than it didn't let me put in my real email address that is tdogjake76@gmail.com  it made me put in @tdogjake76@gmail.com  witch is not my real email address and that lead me hear for the last 4.5 y I've been gaming out majorly and have not received any of my cash cards or cash for all the 100000.00 s of 100000.00 of dollars in game card s that my money was saposto go to my pay pal I am clearey getting scared by all these so called hackers on this profile of mine hear I did not otherize any money rendered to my account to any one I am upset of how can the whole system be scared on every game I play and earn cash cards I have never once got mumy money from all th hundreds of games I downloaded and earned and plays endless amounts of my time to earn those and that cash I've talked it up and approximately 1.8 million is how much gaming I've earned plating game s pls help me stop these hacker from continuing to steel my cash and pls help me get it back thanku for listening i am Tyler Autry Deneault from chase bc Po box 1810 V0E1M0 CANADA tdogjake76@gmail.com 12505742086 pls if u can refund me pls send me a cheuqe in to my mail box thank you

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
